arwen/host/execution.go

```diff
@@ -686,6 +698,10 @@ func (host *vmHost) callInitFunction() error {
 		err = host.handleBreakpointIfAny(err)
```

Comment:
If init() returns err this call could overwrite the init error with nil, it is ok?

Reply:
Yes it is correct